### ISSUE TITLE
RecallWordX outputs additional spaces

### ISSUE THREAD

**Issue by tiymat** [lmorg/murex#808]
**Describe the bug:**
Recalling words using hotkeys on the devel branch results in the words being separated by 2 spaces.

**Expected behaviour:**
Only one space should separate them.

**Screenshots:**
The last line was generated by using the `F1`, `F2`, and `F3` hotkeys in succession.
![image](https://github.com/lmorg/murex/assets/138939221/30ba55a6-4ecb-4b75-b52f-4fd6bdc9139e)

**Platform (please complete the following information):**
 - OS, output from `uname -a` if supported: 6.9.4-arch1-1
 - Terminal Emulator: kitty 0.35.1
 - Murex version, output from `version --no-app-name`: 6.1.7400

**Additional context**
I was able to fix it by removing the `" "` appended to the `rl.insertStr` call at the end of `hkFnRecallWord` and would have submitted a pull request, but given that the space was explicitly added in the first place I assume it is (or was) needed in some context.
![image](https://github.com/lmorg/murex/assets/138939221/f6a4ce99-226d-4abd-ad7d-a94edaf8b938)

**Comment by lmorg** [comment]
> but given that the space was explicitly added in the first place I assume it is (or was) needed in some context.

yeah, you're right. It is needed. As insane as it sounds, the correct way to do this is to trim all the surrounding white space, then re-add a space at the end again. 99% of the time it would be a completely redundant exercise but if you have any instances where the word had multiple spaces, or even somehow a tab character, you'd end up getting that multiple spaces included again.
